File: benchmarks/stage5_vexp_swe_bench_smoke/fixtures/m202_language_families/ruby/valid.rb
# Fixture — café 日本語 before declarations
require "json"

module Fixture
  # A greeter.
  class Greeter
    # Say hello.
    def hello(name)
      "hi #{name}"
    end

    def self.build
      new
    end
  end

  def helper(x)
    x + 1
  end
end

File: benchmarks/stage5_vexp_swe_bench_smoke/fixtures/m202_language_families/lua/valid.lua
-- Fixture — café 日本語 before declarations
local json = require("json")

local M = {}

--- Add two numbers.
function M.add(a, b)
  return a + b
end

--- A local helper.
local function helper(x)
  return x + 1
end

function M.Greeter:hello(name)
  return "hi " .. name
end

return M
